ibm/service/kms/resource_ibm_kms_key_with_policy_overrides_test.go

```diff
@@ -16,9 +17,12 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
-// TODO: Test Has Been broken and is panicking.
-/*
+const ibmKMSSuppressKeyWithPolicyOverridesTests = true
+
 func TestAccIBMKMSKeyWithPolicyOverridesResource_basic(t *testing.T) {
+	if ibmKMSSuppressKeyWithPolicyOverridesTests {
```

Comment:
once you add the registrations to instance policies, remove this code for skipping